README.md

```diff
@@ -33,7 +33,7 @@ You can specify options in `package.json`
 {
   "config": {
     "validate-commit-msg": {
-      "types": ["feat", "fix", "docs", "style", "refactor", "perf", "test", "chore", "revert"], // default
+      "types": ["feat", "fix", "docs", "style", "refactor", "perf", "test", "build", "ci", "chore", "revert"], // default
```

Comment:
This is just documentation for how you might configure things in your own project to add different types. It's not required, but if you wanted to support these types in your project, you could add this to your package.json.